reconciler/reconciler.go

```diff
@@ -140,27 +138,25 @@ outer:
 		case <-refreshTimer.C:
 		}
 
-		now := time.Now()
+		durationUntilRefresh := r.config.RefreshInterval
 
 		// Iterate over the objects in revision order, e.g. oldest modification first.
 		// We look for objects that are older than [RefreshInterval] and mark them for
-		// pending in order for them to be reconciled again.
+		// refresh in order for them to be reconciled again.
 		iter := r.config.Table.LowerBound(r.DB.ReadTxn(), statedb.ByRevision[Obj](lastRevision+1))
 		indexer := r.config.Table.PrimaryIndexer()
+		now := time.Now()
 		for obj, rev, ok := iter.Next(); ok; obj, rev, ok = iter.Next() {
 			status := r.config.GetObjectStatus(obj)
 
+			// The duration elapsed since this was updated.
+			updatedSince := now.Sub(status.UpdatedAt)
```

Comment:
I wonder whether it could be more appropriate to use `time.Since` here, considering that marking an object for refresh is a potentially slow operation due to rate limiting.

Reply:
Ah yes. Good catch!